tests/framework/cli/test_cli.py

```diff
@@ -436,6 +436,23 @@ def fake_run_config(request, fake_root_dir):
         )
         return config_path
 
+    @staticmethod
+    @fixture(params=["run_config.yml", "run_config.json"])
+    def fake_invalid_run_config(request, fake_root_dir):
+        config_path = str(fake_root_dir / request.param)
+        config = {
+            "run": {
+                "pipeline": "pipeline1",
+                "tag": ["tag1", "tag2"],
+                "node-names": ["node1", "node2"],
+            }
+        }
```

Comment:
```suggestion
        config = {
            "run": {
                "pipeline": "pipeline1",
                "tag": ["tag1", "tag2"],
                "node-names": ["node1", "node2"],
            }
            ,"dummy": {"dummy": "dummy"}
        }
```

If I understand correctly the config only read `run`. I test this manually and it seems to work. This is just making sure if user have arbitary configuration kedro shouldn't throw an error.